bigbluebutton-html5/imports/ui/components/app/styles.js

```diff
@@ -49,54 +52,41 @@ const DtfInvert = `
   .tl-container {
     background-color: var(--tl-background) !important;
   }
-  #TD-Tools button, #TD-TopPanel-Undo, #TD-TopPanel-Redo, #TD-Styles {
-    border-color: transparent !important;
-  }
-  [id="TD-StylesMenu"],
-  [id="TD-Styles-Color-Container"],
-  #connectionBars > div
-`;
-
-const DtfBrandingInvert = `
-  body {
-    background-color: var(--darkreader-neutral-background) !important;
-  }
-  header[id="Navbar"] {
-    background-color: var(--darkreader-neutral-background) !important;
-  }
-  section[id="ActionsBar"] {
-    background-color: var(--darkreader-neutral-background) !important;
+  .tlui-slider__thumb {
+    background-color: var(--darkreader-text--color-text-1) !important;
   }
-  select {
-    border: 0.1rem solid #FFFFFF !important;
+  .tlui-button[data-state="hinted"]::after {
+    background-color: var(--darkreader-selection-background) !important;
   }
-  select[data-test="skipSlide"] {
-    border: unset !important;
+  div.tlui-toolbar__inner > div.tlui-toolbar__tools.fade-in {
+    background: var(--darkreader-border--color-selected) !important;
   }
-  div[data-test="presentationContainer"] {
+  div[id="cameraDock"] {
     background-color: var(--darkreader-neutral-background) !important;
   }
-  select {
-    border-top: unset !important;
-    border-right: unset !important;
-    border-left: unset !important;
-  }
-  .tl-container {
-    background-color: var(--tl-background) !important;
-  }
-  #TD-Tools button, #TD-TopPanel-Undo, #TD-TopPanel-Redo, #TD-Styles {
-    border-color: transparent !important;
+  .bnjzQC > div span div:hover {
+    background-color: var(--darkreader-selection-background) !important;
   }
-  [id="TD-StylesMenu"],
-  [id="TD-Styles-Color-Container"],
-  div[data-test="brandingArea"],
   #connectionBars > div
```

Comment:
Some fixes to note background color and caret color
```suggestion
  #connectionBars > div,
  .tl-note__scrim,.tl-arrow-label[data-isediting="true"] > .tl-arrow-label__inner {
    background-color: unset !important;
  }
  textarea {
    caret-color: black !important;
  }
```